config/initializers/rack_attack.rb

```diff
@@ -0,0 +1,71 @@
+class Rack::Attack 
+  LOGIN_ROUTES = %w[/api/auth /dashboard/authenticate].freeze
+
+  default_limit  = ::Settings.rate_limiting.default.limit
+  default_period = ::Settings.rate_limiting.default.period
+  login_limit    = ::Settings.rate_limiting.login.limit
+  login_period   = ::Settings.rate_limiting.login.period
+
+  # Don't enable in test ENV by default
+  @enabled       = false if Rails.env == "test"
+
+  ### Configure Cache ###
+
+  # If you don't want to use Rails.cache (Rack::Attack's default), then
+  # configure it here.
+  #
+  # Note: The store is only used for throttling (not blocklisting and
+  # safelisting). It must implement .increment and .write like
+  # ActiveSupport::Cache::Store
+
+  # Rack::Attack.cache.store = ActiveSupport::Cache::MemoryStore.new 
```

Comment:
We might want to leverage the memcache store (at least on podified)...otherwise since we sit behind a load balancer, the caller will be able to hit `limit * number of ui workers`, since the local store will only apply to a single worker

Reply:
This is an example.  It defaults to `Rails.cache` by default:

https://github.com/rack/rack-attack/blob/12a8390d2dc0160f2966f0793a12ac2ce76bfe35/lib/rack/attack/cache.rb#L10

So this would just be overriding it.

Reply:
Do we want to have a separate section in the settings for configuring the `Rack::Attack.cache`?

Reply:
I can't see why a user would want to configure it separately from the Rails cache - I assume we'd want the same one.

Reply:
Okie doke, then nothing really to change here.

However, should I remove this section just so it doesn't imply there is anything we could have be separate in the future?

Reply:
Removed this section (and most of the other default comments).